utilities/pipelines/sharedScripts/Get-PublishedModuleVersionsList.ps1

```diff
@@ -35,6 +35,10 @@ function Get-PublishedModuleVersionsList {
     try {
         $tagListResponse = Invoke-RestMethod -Uri $tagListUrl
     } catch {
+        if (Test-McrConnection) {
+            Write-Verbose "MCR connection test passed. New modules don't have a published version."
```

Comment:
Sometimes connection errors happen. Not reaching mcr does not  guarantee that version does not exist

Reply:
Nvm I noticed after that the tested link is mcr and not the tag link. Resolving

Reply:
Suggestion:

This function is used in 2 Pester tests. Both are also retrieving the expected next module version by calling Get-ModuleTargetVersion.
If Get-ModuleTargetVersion returns 0.1.0, we know the module is a new one and we don't need to call Get-PublishedModuleVersionsList.